pcp/Platform.c

```diff
@@ -248,6 +248,25 @@ static const char* Platform_metricNames[] = {
    [PCP_METRIC_COUNT] = NULL
 };
 
+#ifndef HAVE_PMLOOKUPDESCS
```

Comment:
I know this works the way specified, but documenting the premise of providing the function if not available upstream would be a nice touch. ;-)

Reply:
Good point, will do.